R/redundant_ifelse_linter.R

```diff
@@ -9,6 +9,35 @@
 #' @param allow10 Logical, default `FALSE`. If `TRUE`, usage like
 #'   `ifelse(x, 1, 0)` is allowed, i.e., only usage like
 #'   `ifelse(x, TRUE, FALSE)` is linted.
+#'
+#' @examples
+#' # will produce lints
+#' lint(
+#'   text = "ifelse(x >= 2.5, TRUE, FALSE)",
+#'   linters = redundant_ifelse_linter()
+#' )
+#'
+#' lint(
+#'   text = "ifelse(x < 2.5, 1L, 0L)",
+#'   linters = redundant_ifelse_linter()
+#' )
+#'
+#' # okay
+#' lint(
+#'   text = "x >= 2.5",
+#'   linters = redundant_ifelse_linter()
+#' )
+#'
+#' lint(
```

Comment:
maybe mention in a comment that this is the strict equivalent, but often just using the logical vector will "just work", something along those lines?

Reply:
nit: use complete sentence syntax (capitalization, terminal `.`)